smart_home/sensors/humidity_sensor.py

```diff
@@ -1,6 +1,10 @@
+from smart_home.logging.logger import setup_logger
 from smart_home.sensors.sensor import Sensor
 
 
 class HumiditySensor(Sensor):
+    logger = setup_logger('HumiditySensor')
```

Comment:
Man könnte vielleicht die Logger-Initialisierung dynamisch halten und statt "HumiditySensor" selbst einzutragen könnte man `HumiditySensor.__name__` verwenden, um den gleichen String zu bekommen.

Reply:
Könnte man aber auch in jeder Klasse benutzen für `setup_logger`

Reply:
kannst lassen